Docker/Dockerfile

```diff
@@ -44,4 +44,4 @@ ENV BP_DB_PATH="/opt/ozw/config/crashes/"
 WORKDIR /opt/ozw/
 EXPOSE 1983
 VOLUME ["/opt/ozw/config/"]
-ENTRYPOINT /usr/local/bin/ozwdaemon -s $USB_PATH -c /opt/ozw/config/ -u /opt/ozw/config/ --mqtt-server $MQTT_SERVER --mqtt-port $MQTT_PORT --stop-on-failure --mqtt-instance $OZW_INSTANCE
+ENTRYPOINT /usr/local/bin/ozwdaemon -s $USB_PATH -c /opt/ozw/config/ -u /opt/ozw/config/ --mqtt-server $MQTT_SERVER --mqtt-port $MQTT_PORT --mqtt-username $MQTT_USERNAME --mqtt-password $MQTT_PASSWORD --stop-on-failure --mqtt-instance $OZW_INSTANCE
```

Comment:
Ironically, I failed to include my only real comment: Shouldn't `$MQTT_xxx` be double quoted to avoid shell-related issues (whitespace, perhaps others as well).

Reply:
Actually we need a “startup script”. Right now if You don’t need any Auth, it still passes the command to ozwdaemon. Need to test if the environment variable is present and then optionally set the command line. I’d like to do that for the other options as well.

Reply:
I did test this which is why I didn't put in defaults for the username and password.  If it's left blank it isn't picked up by the command line parser and is not added to settings.  A startup script is a cleaner approach though.